camayoc/tests/qcs/cli/test_scans.py

```diff
@@ -0,0 +1,332 @@
+# coding=utf-8
+"""Tests for ``qpc scan`` commands.
+
+:caseautomation: automated
+:casecomponent: cli
+:caseimportance: high
+:caselevel: integration
+:requirement: Sonar
+:testtype: functional
+:upstream: yes
+"""
+import io
+import json
+import operator
+import re
+import time
+from pprint import pformat
+
+import pexpect
+import pytest
+
+from .conftest import qpc_server_config
+from .utils import cred_add, source_add
+from camayoc.config import get_config
+from camayoc.constants import BECOME_PASSWORD_INPUT, CONNECTION_PASSWORD_INPUT
+from camayoc.exceptions import (
+    ConfigFileNotFoundError,
+    FailedScanException,
+    WaitTimeError,
+)
+
+
+BY_NAME = operator.itemgetter('name')
```

Comment:
This is mysterious to me. A doc string might be nice. Or maybe different name for the constant.

Reply:
I will add a docstring. I recommend you taking a look on the itemgetter docs to get more information https://docs.python.org/3.5/library/operator.html#operator.itemgetter. The operator attributes are very useful when filtering or sorting lists, for example, the operators are suited to be used as the key argument of sorted.

This is will generate a function that will return the 'name' item value of a dict passed to the BY_NAME.

Also I am considering lowering the case of the variable name since this is not actually a constant. And also finding a good name was very hard, I though on by_name since it is selecting the items where the key is `name` and when passing as the `ids` parameter it seemed good to read like ids=by_name, do you have any suggestion here?